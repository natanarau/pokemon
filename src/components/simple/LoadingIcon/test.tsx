import { render, waitFor, cleanup } from '@testing-library/react';
import { LoadingIcon } from '.';

afterEach(() => {
  cleanup();
  jest.resetModules();
});

describe('<LoadingIcon />', () => {
  test('should render icon correctly', async () => {
    const { getByTestId } = render(<LoadingIcon />)
    await waitFor(() => expect(getByTestId('LoadingIcon')).toBeTruthy())
  })
  test('should match snapshot', () => {
    const { container } = render(
      <LoadingIcon />
    )
    expect(container).toMatchSnapshot();
  });
})