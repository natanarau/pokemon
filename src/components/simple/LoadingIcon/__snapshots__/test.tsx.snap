// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoadingIcon /> should match snapshot 1`] = `
<div>
  <div
    class="ant-spin ant-spin-spinning"
    data-testid="LoadingIcon"
  >
    <span
      class="ant-spin-dot ant-spin-dot-spin"
    >
      <i
        class="ant-spin-dot-item"
      />
      <i
        class="ant-spin-dot-item"
      />
      <i
        class="ant-spin-dot-item"
      />
      <i
        class="ant-spin-dot-item"
      />
    </span>
  </div>
</div>
`;
